code/controllers/subsystem/shuttle.dm

```diff
@@ -374,6 +375,10 @@ SUBSYSTEM_DEF(shuttle)
 		priority_announce("Hostile environment detected. \
 			Departure has been postponed indefinitely pending \
 			conflict resolution.", null, 'sound/misc/notice1.ogg', "Priority")
+		for(var/i in hostileEnvironments)
+			if(istype(i, /mob/living/carbon/alien/humanoid/royal/queen) && !hostileEnvTrackPlayed)
+				play_soundtrack_music(/datum/soundtrack_song/bee/mind_crawler, only_station = TRUE)
+				hostileEnvTrackPlayed = TRUE
```

Comment:
Break out of the loop just in case something wacky happens and this plays twice.
Also `hostileEnvTrackPlayed` variable is never used, only assigned to.

Reply:
It's used right there in the if?

Reply:
oh yea